src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -217,17 +217,18 @@ public ResponseEntity<Void> cutAndPasteNodeSubtree(@PathVariable("studyUuid") UU
         return ResponseEntity.ok().build();
     }
 
-    @PostMapping(value = "/studies/{studyUuid}/tree/subtrees", params = {"subtreeToCopyParentNodeUuid", "referenceNodeUuid"})
+    @PostMapping(value = "/studies/{targetStudyUuid}/tree/subtrees", params = {"subtreeToCopyParentNodeUuid", "referenceNodeUuid"})
     @Operation(summary = "duplicate a subtree")
     @ApiResponses(value = {
         @ApiResponse(responseCode = "200", description = "The subtree was successfully created"),
         @ApiResponse(responseCode = "403", description = "The subtree can't be copied above the root node nor around itself"),
         @ApiResponse(responseCode = "404", description = "The source study or subtree doesn't exist")})
-    public ResponseEntity<Void> duplicateSubtree(@PathVariable("studyUuid") UUID studyUuid,
+    public ResponseEntity<Void> duplicateSubtree(@Parameter(description = "The study where we want to copy the node") @PathVariable("targetStudyUuid") UUID targetStudyUuid,
```

Comment:
MAybe we should keep `{studyUuid}` in the REST path for a resource and just call the variable here with the target prefix
`@PathVariable("studyUuid") UUID targetStudyUuid,`
should be good.

Reply:
Done, everything still works as intended